doc/HOWTO/adding-functions.md

```diff
@@ -0,0 +1,36 @@
+Adding new functions
+====================
+
+This document describes the way to add custom modifications to OpenSSL sources.
+
+ If you are adding new public functions to the custom library build, you need to
+ either add a prototype in one of the existing OpenSSL header files;
+ or provide a new header file and edit
+ [Configurations/unix-Makefile.tmpl](Configurations/unix-Makefile.tmpl)
+ to pick up that file.
```

Comment:
The part about `Configurations/unix-Makefile.tmpl` is *aged*, apart for some very special cases, which should be dealt with on a case by case basis and not worth mentioning here.  So the part "and edit . ...  to pick up that file." can simply be removed.

Reply:
removed in f370ab3